tests/inputFields.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { test, expect } from '@playwright/test';
+
+
+test.describe('Input fields', () => {  
+test.beforeEach( async({page}) => {
+  await page.goto('/')
+  await page.getByRole('link', { name: 'Pet Types' }).click();
+  await page.getByRole('heading', { name: 'Pet Types' }).click()
+})
+
+test('Update pet type', async ({page}) => {
+const tableRow = page.locator('tbody tr')
+await tableRow.getByRole('button', {name: 'Edit'}).first().click()
+await expect(page.getByRole('heading')).toContainText('Edit Pet Type');
+const nameInputField = page.locator('#name')
+await nameInputField.click();
+await nameInputField.clear()
+await nameInputField.fill('rabbit')
+const updateButton = page.getByRole('button', {name: 'Update'})
+await updateButton.click()
+await expect(tableRow.locator('td input[id="0"]')).toHaveValue('rabbit')
+await tableRow.getByRole('button', {name: 'Edit'}).first().click()
+await nameInputField.click();
+await nameInputField.clear()
+await nameInputField.fill('cat')
+await updateButton.click()
+await expect(tableRow.locator('td input[id="0"]')).toHaveValue('cat')
+
+});
+
+test('Cancel pet type update', async ({ page }) => {
+  const editDogRow = page.getByRole('button', { name: 'Edit'}).nth(1);
+  await editDogRow.click()
```

Comment:
Two problems here:
1. Incorrect constant name. The more correct name would be something like `secondEditButton` or something like this. Imagine you change the name of the pet for this row, and it will not be a dog anymore, so the constant name will not match it's implementation
2. This constant was not used anywhere else in the code, so you can write lines 32 and 33 just a single line expression
`await page.getByRole('button', { name: 'Edit'}).nth(1).click()`